openshift/nginx.conf

```diff
@@ -131,18 +132,18 @@ http {
         }
 
         # Media: images, icons, video, audio, HTC
-        location ~* \.(?:jpg|jpeg|gif|png|ico|cur|gz|svg|svgz|mp4|ogg|ogv|webm|htc)$ {
-          expires 1M;
-          access_log off;
-          add_header Cache-Control "public";
-        }
-
-        # Javascript and CSS files
-        location ~* \.(?:css|js)$ {
-            try_files $uri =404;
-            expires 1y;
-            access_log off;
-            add_header Cache-Control "public";
-        }
+        # location ~* \.(?:jpg|jpeg|gif|png|ico|cur|gz|svg|svgz|mp4|ogg|ogv|webm|htc)$ {
```

Comment:
Do we need these commented out lines?

Reply:
I'm just going to pull out nginx entirely. It really doesn't serve much purpose at the moment because the it's using redis.